File: accesshqWebTesting/src/test/java/com/accesshq/strategies/RediusMatchingStrategy.java
package com.accesshq.strategies;

import com.accesshq.model.Planet;

import java.text.ParseException;

public class RediusMatchingStrategy implements MatchingStrategy{

    private double number;
    public RediusMatchingStrategy(double number) {
        this.number = number;
    }

    @Override
    public boolean match(Planet planet) throws ParseException {
        return planet.getRadius().equals(number);
    }
}

File: accesshqCsharp/CodeCampCsharp/CodeCampCsharp/Strategyies/RediusMatchingStrategy.cs
using CodeCampCsharp.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CodeCampCsharp.Strategyies
{
    public class RediusMatchingStrategy : MatchingStrategy
    {
        private double number;
        public RediusMatchingStrategy(double number)
        {
            this.number = number;
        }

        public bool Match(Planet planet)
        {
            return planet.GetRadius().Equals(number);
            throw new NotImplementedException();
        }
    }
}
